pinot-server/src/main/java/com/linkedin/pinot/server/starter/ServerBuilder.java

```diff
@@ -69,9 +70,19 @@ private void initMetrics() {
     _serverMetrics.initializeGlobalMeters();
   }
 
+  @SuppressWarnings("unchecked")
   private void initTransformFunctions() {
-    TransformFunctionFactory.init(
-        ArrayUtils.addAll(TransformUtils.getBuiltInTransform(), _serverConf.getTransformFunctions()));
+    Set<Class<TransformFunction>> transformFunctionClasses = new HashSet<>();
+    for (String transformFunctionClassName : _serverConf.getTransformFunctions()) {
+      Class<TransformFunction> transformFunctionClass;
+      try {
+        transformFunctionClass = (Class<TransformFunction>) Class.forName(transformFunctionClassName);
+      } catch (ClassNotFoundException e) {
+        throw new RuntimeException("Failed to find transform function class of name: " + transformFunctionClassName);
+      }
+      transformFunctionClasses.add(transformFunctionClass);
+    }
+    TransformFunctionFactory.init(transformFunctionClasses);
```

Comment:
check if the transform function already exists

Reply:
Done